scripts/screens/ProfileScreen.py

```diff
@@ -263,12 +263,12 @@ def handle_tab_events(self, event):
             if event.ui_element == self.change_name_button:
                 ChangeCatName(self.the_cat)
             elif event.ui_element == self.specify_gender_button:
-                SpecifyCatGender(self.the_cat)
-                '''if self.the_cat.genderalign in ["female", "trans female"]:
+                self.change_screen("change gender screen")
+                if self.the_cat.genderalign in ["female", "trans female"]:
```

Comment:
This is currently the origin of a bug,where going to the cat's gender specify page will re-set their pronouns to correspond with their gender, clearing any previously set pronouns and ignoring the they/them default setting. Remove lines 266-271 to fix this issue.

Reply:
Whoops, good catch, I remember thinking to myself a few months back I need to fix that, guess I forgot XD